test/libsolidity/SemVerMatcher.cpp

```diff
@@ -159,11 +175,19 @@ BOOST_AUTO_TEST_CASE(positive_range)
 	for (auto const& t: tests)
 	{
 		SemVerVersion version(t.second);
-		SemVerMatchExpression expression = parseExpression(t.first);
-		BOOST_CHECK_MESSAGE(
-			expression.matches(version),
-			"Version \"" + t.second + "\" did not satisfy expression \"" + t.first + "\""
-		);
+		try
+		{
+			SemVerMatchExpression matchExpression = parseExpression(t.first);
+			BOOST_CHECK_MESSAGE(
+				matchExpression.matches(version),
+				"Version \"" + t.second + "\" did not satisfy expression \"" + t.first + "\""
+			);
+		}
+		catch(SemVerError const& matchError)
```

Comment:
Surprised the compiler is not complaining against the unused `matchError`.

```suggestion
		catch(SemVerError const&)
```

Reply:
This was my bad. I committed this without a proper check.

Reply:
While you're at it, can you add a comment like `// Ignored` in the `catch` block as well?